model/schema/expression.yaml

```diff
@@ -12,14 +12,33 @@ imports:
 
 classes:
 
-  AnatomicalSite:
+  ExpressionExperiment:
+    mixins:
+      - AuditedObject
     description: >-
-      The developmental stage and/or age of the specimen in an annotation.
+      Defined by the gene of interest, the specimen, the assay, the reagents 
+      (Antibody, Probe), and the reference. It groups ExpressionAnnotations.
+    notes: >-
+      Specimens are defined by their AGM and/or alleles (or gel lane at MGI).
     slots:
-      - anatomical_structure
-      - anatomical_substructure
-      - cellular_component
-      - anatomical_qualifiers
+      - curie
+      - references
+      - biological_entity_assayed
+      - assay_used
+      - reagents_used
+      - specimen_genomic_model
+      - specimen_alleles
+      - condition_relations
+    slot_usage:
+      references:
+        multivalued: false    # Should I call this slot just "reference" then?
```

Comment:
We need a name for reference when it is single-valued, which is different than 'Reference' (the name of the Class).  What about 'attribution'?